### ISSUE TITLE
Support for Xiaofang 1s? 

### ISSUE THREAD

**Issue by newsera** [samtap/fang-hacks#341]
https://en.xiaomitoday.it/xiaofang-1s-e-la-nuova-ip-camera-di-xiaomi.html

Does this hack script support the newer Xiaofang 1s?

**Comment by SamHead** [comment]
Is the Xiaofang 1s same hardware as WyzeCam V2 ? https://www.wyzecam.com/product/wyze-cam-v2/

**Comment by mouth4war** [comment]
Yes, it is the same hardware. https://github.com/EliasKotlyar/Xiaomi-Dafang-Hacks
